sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -531,7 +674,25 @@ private void applyForcedConfigFromStrategy(
     consumerConfig.setConfig(newConfig);
   }
 
-  /** Initial checks. Configuration mus be initialized */
+  private static Serializer<?> getSerializer(Class<?> clazz, ObjectMapper objectMapper) {
```

Comment:
```suggestion
  private static Serializer<T> getSerializer(Class<T> clazz, ObjectMapper objectMapper) {
```

I didn't tested it, but I would assume that we could get rid of the unchecked warnings above.